changed line2 of 
/usr/lib/python2.6/dist-packages/scipy/interpolate/polyint.py
FROM:
from scipy import factorial
TO:
from scipy.misc.common import factorial


--verbose-helpful



Still have to fix matplotlb datapath
Explorar relacion de files de matplotlib y py2exe


Modications to /usr/lib/pymodules/python2.6/matplotlib/__init__.py
changed linne 459.
This would deal with the datapath issue
 #### LINE MODIFIED FOR SUPPORT MIX
     elif os.path.isdir('mpl-data'): return 'mpl-data'
 ### END MODIFICATION   
